### PR TITLE
Submission for zero.is-a.dev

### PR DESCRIPTION
**Pull request by CoderPopCat**
I have created `zero.json` as info for adding the domain

### CODE REVIEW

File: domains/zero.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "My personal developer portfolio.",
+  "repo": "https://github.com/CoderPopCat/coderpopcat.github.io",
+  "owner": {
+    "username": "CoderPopCat",
+    "email": "zerotwo@popcat.xyz",
+    "twitter": "none"
```
Comment (MaskDuck) [inline domains/zero.json:7]: is `twitter.com/none` your twitter account?

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/zero.json (hunk 1 above)
Comment (CoderPopCat) [reply domains/zero.json:7]: it means I have no twitter account

Comment (MaskDuck) [reply domains/zero.json:7]: then why are you filling it

Comment (MaskDuck) [reply domains/zero.json:7]: if you dont have any twitter just omit it

**Review by MaskDuck — APPROVED** [review]
sus

**Comment by CoderPopCat** [comment]
uhh I can't see a 404 page on my domain

**Comment by CoderPopCat** [comment]
> uh setup it. find how to [here](https://github.com/is-a-dev/register/tree/master/docs/hosted-at/github-pages.md)

it is done

**Comment by MaskDuck** [comment]
also on ann unrelated note dont forget to leave a star lol

**Comment by CoderPopCat** [comment]
> also on ann unrelated note dont forget to leave a star lol

ok

**Comment by CoderPopCat** [comment]
still nothing on the page and its not even connected

**Comment by MaskDuck** [comment]
can you create an issue?

**Pull request merged**